src/modules/abtest/finish.ts

```diff
@@ -21,6 +21,7 @@ const promptWorkspaceToFinishABTest = () =>
   abtester
     .status()
     .then(map(({ WorkspaceB }) => WorkspaceB))
+    .then(filter((workspace: any) => workspace !== undefined))
```

Comment:
```suggestion
    .then(filter((workspace: string | undefined) => workspace !== undefined))
```

Reply:
Have you tried to apply this change? It seems we would need to add `// @ts-ignore` to the preceding line.
<img width="764" alt="Captura de Tela 2021-04-12 às 12 14 00" src="https://user-images.githubusercontent.com/67928158/114419440-d0bffd00-9b89-11eb-9c6e-7c03db515f14.png">

Reply:
Yes, you are correct. Ideally the `abTester` client would have appropriate typings so the function caller wouldn't have to deal with these issues. One workaround for not having these types in place is this:

```
const promptWorkspaceToFinishABTest = () =>
  abtester
    .status()
    .then(filter(({ WorkspaceB }) => WorkspaceB !== undefined))
    .then(map(({ WorkspaceB }) => WorkspaceB))
    .then((workspaces: string[]) =>
      enquirer.prompt<{ workspace: string }>({
        name: 'workspace',
        message: 'Choose which workspace to finish A/B testing:',
        type: 'select',
        choices: workspaces,
      })
    )
    .then(prop('workspace'))

```

Reply:
Nice! Maybe we should add the typings to the client. I'll take a quick look at that.

Reply:
I figured it makes more sense to do that in another PR, as you have already found a workaround.